siliconcompiler/schema/schema_obj.py

```diff
@@ -570,7 +570,7 @@ def _normalize_value(value, sc_type, error_msg, allowed_values):
 
         if sc_type in ('file', 'dir'):
             if isinstance(value, (str, pathlib.Path)):
-                return str(value)
+                return pathlib.Path(value).as_posix()
```

Comment:
we should still store it as a string (otherwise way too many things might break)

Reply:
`.as_posix()` returns a string, not a Path.